scripts/packr-scripts/linux/linux-pack.sh

```diff
@@ -0,0 +1,69 @@
+#!/bin/bash
+
+set -e
+set -u
+set -o pipefail
+
+SCRIPT_DIR=$(realpath $(dirname $0))
+PARENT_DIR=$(realpath ${SCRIPT_DIR}/..)
+source ${PARENT_DIR}/common-env-variables.sh
+
+cd ${SCRIPT_DIR}
+JRE_ZIP_FILE_NAME=$(ls jre*.zip) #jre1.8.0_152.zip
+RELEASE_VERSION=$(git describe --abbrev=0 --tags | tr -d v)
+OS_NAME=linux
+RECORD_AND_UPLOAD_JAR=${PACKAGE_NAME}-${OS_NAME}-${RELEASE_VERSION}.jar
+TGZ_ARCHIVE_NAME=$(getArchiveName "${OS_NAME}-${RELEASE_VERSION}" "tgz")
+
+if [[ ! -s ${JRE_ZIP_FILE_NAME} ]]; then
```

Comment:
Yes I missed that one, good call again. Can you please make them available in a non-public way and give me the name of the environment variable which will contain the URL.
so for e.g.

```bash
wget ${JRE_LINUX}
wget ${JRE_MACOS}
wget ${JRE_WINDOWS}
```

It is private to the build process so not end-user dependent, it can be anything you like. I would use the above or an improvised version of it.

Reply:
Fixed by 66ac680